src/request.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-console */
+'use strict';
+
+const http = require('http');
+
+const BASE = 'http://localhost:8800';
+const pathname = '/file/index.html';
+const href = BASE + pathname;
+
+const req = http.request(href, (res) => {
+  console.log(res.statusCode);
```

Comment:
You are only logging the status code of the response. It would be better to handle different status codes differently. For example, if a 404 status code is returned, you could return a message saying the file was not found.